plugins/jira-dashboard/src/components/JiraDashboardContent/JiraDashboardContent.tsx

```diff
@@ -23,7 +23,10 @@ import { useJira } from '../../hooks/useJira';
 import { isJiraDashboardAvailable } from '../../plugin';
 import { JiraDataResponse } from '@axis-backstage/plugin-jira-dashboard-common';
 
-export const JiraDashboardContent = (props?: { annotationPrefix?: string }) => {
+export const JiraDashboardContent = (props?: {
+  annotationPrefix?: string;
+  showFilters?: boolean;
```

Comment:
Why not make which filters to use configurable? The props could take the filters to use? 

```
export const JiraDashboardContent = (props?: {
  annotationPrefix?: string;
  tableFilters?: TableFilter;
}) => {
```

Reply:
The filters were originally configurable but this was changed to optionality on feedback from @fridajac. 
This also prevents users from enabling the assignee filter which will need further contribution.

Reply:
Can't we satisfy both? We can have the "tableFilters" props, that can take in TableFilter - like @anicke showed, or a boolean. If it's just boolean = true, then the default filters are used? 
```
export const JiraDashboardContent = (props?: {
  annotationPrefix?: string;
  tableFilters?: TableFilter | boolean;
}) => {
```

Reply:
I updated the arguments to support this functionality. Additionally since users would now have the option to pass in the assignee field as an option, I added a function to handle unassigned users, otherwise the null value would cause the table to display an error when clicking the filter icon.
![image](https://github.com/user-attachments/assets/f422844e-3206-4900-9dac-ac36d4158c6d)